src/components/timeseriesexplorer.js

```diff
@@ -20,6 +20,70 @@ const TimeSeries = lazy(() =>
   import('./timeseries' /* webpackChunkName: "TimeSeries" */)
 );
 
+function Chart({
+  isVisible,
+  stateCode,
+  timeseries,
+  dates,
+  chartType,
+  isUniform,
+  isLog,
+}) {
+  return (
+    isVisible && (
+      <Suspense fallback={<TimeseriesLoader />}>
+        <TimeSeries
+          {...{timeseries, dates, chartType, isUniform, isLog, stateCode}}
+        />
+      </Suspense>
+    )
+  );
+}
+
+function Pills(props) {
```

Comment:
Can you destruct props in the method signature itself just like you did for the `Charts` component?
```suggestion
function Pills({ timeseriesOption, ...so..on }) {
```

Reply:
Sure, will do that!